package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@fastify/workflows",
-  "version": "3.9.1",
+  "version": "3.9.2",
```

Patch to `v3.9.3` as there is already `v3.9.2`
```suggestion
  "version": "3.9.3",
```